debezium-connector-sqlserver/src/main/java/io/debezium/connector/sqlserver/SqlServerConnectorConfig.java

```diff
@@ -44,6 +44,7 @@
 
     private static final Logger LOGGER = LoggerFactory.getLogger(SqlServerConnectorConfig.class);
 
+    public static final String SKIP_LSN_TIMESTAMP_QUERY_CONFIG_NAME = "skip.lsn.timestamp.query";
```

Comment:
I'd suggest to name this option more after the semantics of the timestamp. Perhaps "source.timestamp.mode" = {commit|processing}. I.e. we wouldn't expose technicalities like how the value is retrieved (e.g. this might change in a future SQL Server version where we could get the value without such query, but its semantics of being commit-LSN-based would still be the same). Also it'd open room for adding other strategies later on, if needed.

Reply:
This makes all the sense to me. Honestly, I was not really happy with the property name I suggested since it is quite specific on how it was implemented, without having a clear semantic. I will use your suggestion! Thanks.